packages/@sanity/form-builder/src/inputs/Array/ItemValue.js

```diff
@@ -107,7 +107,7 @@ export default class Item extends React.Component<Props> {
 
     return (
       <div className={styles.popupAnchor}>
-        <EditItemPopOver title={memberType.title} onClose={this.handleEditStop}>
+        <EditItemPopOver onClose={this.handleEditStop} key={`popOver${item._key}`}>
```

Comment:
No need to prefix the key here as long as it is unique within its parent, so its cleaner to just do `key={item._key}` here :)